### PR TITLE
Updates the CloudSensei Subject info

### PR DESCRIPTION
**Pull request by krishvoor**
This PR updates the Subject info to reflect consistency in email reporting.

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi 👋 @ebattat @athiruma ptal,

**Review by ebattat — APPROVED** [review]
/LGTM

File: cloudsensei/agg_lambda/lambda_function.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def lambda_handler(event, context):
     message = "Something went wrong check your es_data"
     es_operations = ESOperations()
     email_body = ""
-    subject = "Daily Report of long running instances in Perf&Scale AWS Accounts"
```
Comment (athiruma) [inline cloudsensei/agg_lambda/lambda_function.py:25]: Weekly Cloud Report: Long-running instances in the Perf&Scale AWS Accounts?

Comment (krishvoor) [reply cloudsensei/agg_lambda/lambda_function.py:25]: Updated.

File: cloudsensei/lambda_function.py [hunk 2]
```diff
@@ -187,7 +187,7 @@ def lambda_handler(event, context):
     extra_message = ''
     ec2_operations = EC2Operations()
     account_name = ec2_operations.get_account_alias_name()
-    process_data = ProcessData(subject=f'Daily Report @{account_name} has following long running instances')
```
[the remark this answers is not in the document]
Comment (krishvoor) [reply cloudsensei/lambda_function.py:190]: Updated.

**Review by athiruma — APPROVED** [review]
\LGTM

**Pull request merged**